charts/victoria-metrics-k8s-stack/hack/sync_rules.py

```diff
@@ -158,33 +178,70 @@ def write_group_to_file(group, url, destination):
     makedirs(destination, exist_ok=True)
 
     # recreate the file
-    with open(new_filename, 'w') as f:
+    with open(new_filename, "w") as f:
         f.write(escape(lines))
 
     print(f"Generated {new_filename}")
 
 
+def jsonnet_import(directory, file_name):
+    src = join(directory, file_name)
+    if file_name.startswith("github.com"):
+        parsed = urlparse("https://%s" % file_name).path.split("/", 3)
+        src = "https://raw.githubusercontent.com/%s/%s/master/%s" % (
+            parsed[1],
+            parsed[2],
+            parsed[3],
+        )
```

Comment:
Could you add a comment to clarify which cases are fixed by this?
As far as I can see, it is most likely related to jsonnet being able to reference other files in github repo directly, but it would be great to get this noted.

Also, is it ok to hardcode `master` branch here?
Maybe it is possible to fetch original branch name from the `file_name` in order to avoid issues in case branch will be renamed in upstream.

Reply:
jsonnet has remote data directives in a format, like in go (check [here](https://github.com/etcd-io/etcd/blob/main/contrib/mixin/dashboards/g.libsonnet)). this code invoked by jsonnet on each met import and converts imports to exact data from remote sources (jsonnet lib does not perform it by default)

Reply:
> Also, is it ok to hardcode master branch here?
Added api call to get a default branch name